Completed task b-1 Get all movies:

Anur13 opened the pull request:
- added jackson-datatype-jsr310 to convert LocalDateTime of Movie objects to required format when responding with JSON
- added unit test for Movie mapper
- added integration tests that check whether the response is in JSON format and check that at least one movie is present

Anur13 committed:
Completed task b-1 Get all movies:
- added jackson-datatype-jsr310 to convert LocalDateTime of Movie objects to required format when responding with JSON
- added unit test for Movie mapper
- added integration tests that check whether the response is in JSON format and check that at least one movie is present

Rooman commented on src/main/java/com/andrew/movieLand/web/controller/MovieController.java:
```diff
@@ -0,0 +1,25 @@
+
+import java.util.List;
+
+@Controller
```
@RestController

Rooman commented on src/main/resources/spring/app-servlet.xml:
```diff
@@ -0,0 +1,37 @@
+        <property name="dataSource" ref="dataSource"/>
+    </bean>
+
+    <bean id="JdbcMovieDao" class="com.andrew.movieLand.dao.jdbc.JdbcMovieDao"/>
```
Create with anotaion @REpository

Anur13 committed:
Refactoring:
- Improved modifiers, names, formatting across mutiple classes
- Added hamcrest, json-path, slf4j, Hikari
- Split app-servlet.xml into controller.xml and applicationContext.xml
- Improved Movie controller test
- Added movie controller test with mock however not sure if it's ok or how to do better

Anur13 committed:
removes target from commit

Anur13 committed:
added flyway and migrations for movies table creation and fill
